whisper_live/client.py

```diff
@@ -344,6 +345,46 @@ def write_audio_frames_to_file(self, frames, file_name):
             wavfile.setframerate(self.rate)
             wavfile.writeframes(frames)
 
+    def process_hls_stream(self, hls_url):
+        """
+        Connect to an HLS source, process the audio stream, and send it for transcription.
+
+        Args:
+            hls_url (str): The URL of the HLS stream source.
+        """
+        print("[INFO]: Connecting to HLS stream...")
+        process = None  # Initialize process to None
+
+
+        try:
+            # Launch an FFMPEG process to connect to the HLS stream
+            command = [
+                'ffmpeg',
```

Comment:
With this in place, we also should mention in the README that ffmpeg is a requirment.

Reply:
I've now changed the requirement of having ffmpeg installed by using the `ffmpeg-python` library. I've now resubmitted a new version of the code. Please check.